src/utils/dateUtils.ts

```diff
@@ -41,3 +43,13 @@ export const LocalTimeToJsonPlugin = (options: never, dayjsClass: typeof Dayjs):
         return this.format('YYYY-MM-DDTHH:mm:ss');
     };
 };
+
+/**
+ * Dayjs global configuration
+ */
+export const dayjsConfig = (lang = 'en'): void => {
```

Comment:
@djbusstop Hey! On adding the dayjs extension for using things like `fromNow()` I also realised this needs to be localised. This is done globally but needs to know which locale to actually use. I tried dynamic imports but couldn't make it work, so for now the solution is to preload all the available locales (currently only en & sv) and then select one once `lang` arrives with pageProps. I also pulled the dayjs config together into one place just for neatness.
If you can think of a better way of doing this then let me know x

Reply:
I think this is fine for now. If we ever have problems with bundle sizes, we could perhaps build different bundles for each supported locale. But doing anything like that now would be premature optimization I reckon.